vllm_spyre/platform.py

```diff
@@ -128,8 +128,12 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
             os.environ["COMPILATION_MODE"] = "offline"
 
         if envs_spyre.VLLM_SPYRE_USE_CB and is_decoder:
-            scheduler_config.scheduler_cls = "vllm_spyre.v1.core."\
-                "scheduler.ContinuousBatchingSpyreScheduler"
+            if envs_spyre.VLLM_SPYRE_USE_CHUNKED_PREFILL:
```

Comment:
If we don't already, we should probably either:
1. Set `VLLM_SPYRE_USE_CB=1` if `VLLM_SPYRE_USE_CHUNKED_PREFILL` is set, or
2. Raise an error if `VLLM_SPYRE_USE_CHUNKED_PREFILL` is set and `VLLM_SPYRE_USE_CB` is not